src/pages/patientView/clinicalInformation/PatientViewPageStore.spec.ts

```diff
@@ -33,4 +35,36 @@ describe('ClinicalInformationSamplesTable', () => {
         assert.deepEqual(result,[]);
     });
 
+    it('won\'t fetch cosmic data if there are no mutations', ()=>{
```

Comment:
these test are not comprehensive but they catch the key thing, that we handle case where no mutations are present.